packages/edit-site/src/components/editor/index.js

```diff
@@ -349,7 +342,16 @@ export default function Editor( { isLoading, onClick } ) {
 							/>
 						}
 						content={
-							<>
+							<EditorProvider
```

Comment:
So I didn't dive yet into the issue to understand it properly but for me, conceptually at least, `EditorProvider` should wrap the whole `InterfaceSkeleton`, it provides the context for the canvas, the header and the sidebar. The fact that the header and sidebar continue to work with this change is just because we're doing the `useSubRegistry={ false }` hack to allow third-party developers to access the core/editor store globally, without it, it would break both of these components as they won't be "talking" to the right store.

Reply:
Ok here's an alternative fix https://github.com/WordPress/gutenberg/pull/61882